app/controllers/observations_controller.rb

```diff
@@ -6,7 +6,7 @@ class ObservationsController < ApplicationController
 
   def create
     obs = @proposal.add_observer(observer_email, current_user, params[:observation][:reason])
-    flash[:success] = "#{obs.user.full_name} has been added as an observer"
+    prep_create_response_msg(obs)
```

Comment:
what about `observer` as a var name here (and below) rather than `obs` ? -- would be clearer, imo!